qiskit/primitives/statevector_sampler.py

```diff
@@ -52,6 +52,80 @@ class _MeasureInfo:
 class StatevectorSampler(BaseSamplerV2):
     """
     Simple implementation of :class:`BaseSamplerV2` using full state vector simulation.
+
+    This class is implemented via :class:`~.Statevector` which turns provided circuits into
+    pure state vectors, and is therefore incompatible with mid-circuit measurements (although
+    other implementations may be).
+
+    As seen in the example below, this sampler supports providing arrays of parameter value sets to
+    bind against a single circuit.
+
+    Each tuple of ``(circuit, <optional> parameter values, <optional> shots)``, called a sampler
+    primitive unified bloc (PUB), produces its own array-valued result. The :meth:`~run` method can
+    be given many pubs at once.
+
+    .. code-block:: python
+
+        from qiskit.circuit import Parameter, QuantumCircuit, ClassicalRegister, QuantumRegister
```

Comment:
Like with other examples width is probably a bit too large in places for rendering without a lot of side scrolling